STM32/LightController/Core/Src/stm32f4xx_hal_msp.c

```diff
@@ -93,32 +91,13 @@ void HAL_TIM_Base_MspInit(TIM_HandleTypeDef* htim_base)
   /* USER CODE END TIM2_MspInit 0 */
     /* Peripheral clock enable */
     __HAL_RCC_TIM2_CLK_ENABLE();
+    /* TIM2 interrupt Init */
+    HAL_NVIC_SetPriority(TIM2_IRQn, 0, 0);
+    HAL_NVIC_EnableIRQ(TIM2_IRQn);

```

Comment:
The priority is the same as TIM5. I think it would be better if you could work out which was higher priority

Reply:
Fair point. I've set the switching timer to have higher priority.